ray/deployments/base.py

```diff
@@ -1,24 +1,35 @@
+import asyncio
 import gc
-from concurrent.futures import ThreadPoolExecutor, Future, TimeoutError
-from functools import partial, wraps
+import traceback
+import weakref
+from concurrent.futures import Future, ThreadPoolExecutor, TimeoutError
+from functools import wraps
 from typing import Any, Dict
 
 import ray
+import socketio
 import torch
 from minio import Minio
 from pydantic import BaseModel, ConfigDict
 from torch.amp import autocast
-from torch.cuda import (max_memory_allocated, memory_allocated,
-                        reset_peak_memory_stats)
+from torch.cuda import (
```

Comment:
It seems like you have specific preferences on how to format things (using parentheses, vertically listing arguments, using double quotes) which differs from how I do things normally. Maybe we should have a custom linter to standardize things and keep the code style consistent?